frontend/libs/studio-components/src/components/StudioSearch/StudioSearch.module.css

```
@@ -0,0 +1,3 @@
+.studioSearch [id^='search-error'] {
+  display: none;
+}
```

Vi bør så langt det lar seg gjøre unngå å bruke egenskaper fra designsystemet i selektorene. De kan endre seg. Komponenten bør også være så isolert som mulig. Det vil si at selv om vi vet at vi ikke viser noen feilmeldinger der den er i bruk nå, skal ikke komponenten være avhengig av den kunnskapen.

Her er en tryggere måte å løse dette på:
```suggestion
.studioSearch div:empty {
  display: contents;
}
```
Vi kunne også brukt `display: none`, men det kan føre til at enkelte tilgjengelighetsverktøy ikke oppfatter feilmeldingen når den dukker opp.

Dette er uansett ikke en helt ideell løsning fordi den fortsatt baserer seg på implementasjonsdetaljer i Designsystemet. Og dette ser ut til å stamme nettopp derfra. Det mest ryddige ville derfor vært å løse det der. Har du vurdert å opprette en sak for designsystemteamet?

---

Skjønner. Vi kan godt legge til koden midlertidig, men da må vi også ha en `Todo`-kommentar som sier at vi kan fjerne den når designsystemet er oppdatert, helst med en referanse til en sak der dette er diskutert, hvis det er tilgjengelig.